block/internal/common/expected_interfaces.go

```diff
@@ -7,8 +7,9 @@ import (
 	"github.com/celestiaorg/go-header"
 )
 
-// broadcaster interface for P2P broadcasting
+// Broadcaster interface for P2P broadcasting
 type Broadcaster[H header.Header[H]] interface {
 	WriteToStoreAndBroadcast(ctx context.Context, payload H, opts ...pubsub.PubOpt) error
 	Store() header.Store[H]
+	SyncHead() (H, error)
```

Comment:
This will tell you what the latest view of network head is BTW (it's the same as listening from gossipsub).